src/Chainweb/Pact/PactService.hs

```diff
@@ -1006,7 +1006,7 @@ execValidateBlock currHeader plData = do
     -- TODO: knob to configure whether this rewind is fatal
     fatalRewindError a h1 h2 = do
         -- TODO: improve error message, give instructions
-        let msg = concat [ "Fatal error: rewind limit exceeded: "
+        let msg = mconcat [ "Fatal error: rewind limit exceeded: "
```

Comment:
I don't see the point behind this change (and now the indent is wrong)

Reply:
This comment is about an outdated piece of code.